proposals/20230511-roadmap-management.md

```diff
@@ -0,0 +1,96 @@
+
+Maintainers will meet in [planning sessions](###roadmap-planning) to update the roadmap with the latest information.
+
+
+### Roadmap Planning
+
+The roadmap serves as a strategic planning tool that outlines the goals and objectives for Falco. Its purpose is to visually represent the overall direction and timeline, enhance transparency and engage the community.
+
+The onus is on the [Core Maintainers](https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md#core-maintainers) to manage the roadmap. In this regard, Core Maintainers meet in **planning sessions on the first week of each calendar month**. 
+
+During these planning sessions, tasks are allocated to the current iteration or postponed to one of the following iterations. The assigned iteration indicates the projected completion date for a particular workstream.
+
+When a session matches with the commencement of an iteration, maintainers convene to assess the planning and prioritize tasks for the iteration. The first planning session of a release cycle must define top priorities for the related release.
+
+TODO: use GitHub labels to propose items for the roadmap
+
+## Testing and Quality Assurance (QA)
+
+Each iteration's output must include at least one Falco pre-release (or a viable development build) designated for testing and QA activities. While it's acceptable for these builds to contain unfinished features or known bugs, they must enable any community member to contribute to the testing and QA efforts.
+
+The targeted schedule for these Testing/QA activities should be the **last week of each iteration** (or earlier during the *Release Preparation*).
+
+Testing and Quality Assurance criteria and procedures must be defined and documented across relevant repositories.
+
+Furthermore, given the strong reliance of Falco on [falcosecurity/libs](https://github.com/falcosecurity/libs), the above-mentioned pre-release/build for Testing/QA purposes must be based on the most recent *libs* development for the intended iteration. This means that during each interaction, a *libs* release (either pre or stable) must happen early enough to be used for this purpose.
+
+## Next Steps and Conclusions
+
+The Falco 0.36 release cycle, running from June to September 2023, will mark the initiation of the new process. This cycle will also serve as an experimental phase for refining the process.
+
+Furthermore, as soon as possible we will kick off a Working Group specifically for this purpose to ensure a smooth execution. This group will involve community members in assisting maintainers with roadmap management. The Working Group's responsibilities will include:
+
+- Address input from the [2023-04-27 Core Maintainers meeting](https://github.com/falcosecurity/community/blob/main/meeting-notes/2023-04-27-Falco-Roadmap-Discussion.md)
+- Sorting and reviewing pending issues to identify key topics for discussion and potential inclusion in the roadmap
```

Comment:
After looking at this, I have an idea about the potential output of this WG.

Essentially, from the proposal it looks like it'll still be core maintainers' job to accept or reject features for the roadmap (which is right), but how can the WG help them making those decisions? It could make sense to constantly produce as part of the WG a list of suggested features to add/prioritize/deprioritize in the roadmap along with reasons why (why is this important for the community etc). Then, core maintainers would review that list and ideally accept the suggestions, but the bulk of the work would already be done by the WG. For this to work right it'll be beneficial to have members of the WG that are very close to the community to provide insights in our users' needs, regardless of whether they write code or not.

Reply:
I've tried to better explain that. Thank you!